src/lib/deploy.js

```diff
@@ -495,30 +495,16 @@ export async function deploy({ alias, yes }) {
 
 // Get the specified blockchain explorer url with txn hash
 function getTxnUrl(graphQLUrl, txn) {
-  const MINASCAN_BASE_URL = `https://minascan.io/berkeley/zk-transaction/`;
-  const MINA_EXPLORER_BASE_URL = `https://berkeley.minaexplorer.com/transaction/`;
-  const explorers = [MINASCAN_BASE_URL, MINA_EXPLORER_BASE_URL];
-  const randomExplorersIndex = Math.floor(Math.random() * explorers.length);
-
   const explorerName = new URL(graphQLUrl).hostname
     .split('.')
     .filter((item) => item === 'minascan' || item === 'minaexplorer')?.[0];
```

Comment:
This was originally used to determine the explorer name so the correct base url could be shown to the user. If we are only showing minascan urls, then this can be removed.

Reply:
Thank you @ymekuria! 
I actually was thinking to leave it but forgotten to rename the variable to something like `broadCastService` because we probably want to show minascan link only when users were using public (and known) broadcast service and just show txn hash for any other cases.
What do you think?

Reply:
I like the idea!

Reply:
Resolving then.